js/components/selection-checkbox.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
+import pureRender from 'pure-render-decorator'
+import Answer from './answer'
+
+import '../../css/question.less'
+
+@pureRender
+export default class SelectionCheckbox extends Component {
+  constructor(props) {
+    super(props)
+  }
+
+
+  onChange(evt) {
+    const {questionKey, onSelectChange} = this.props
+    onSelectChange(questionKey, evt.target.checked)
+  }
+
+  render() {
+    const { selected, hideControls } = this.props
+    if (hideControls) {
+      return null
+    }
+    return(<input type='checkbox' checked={selected} onChange={this.onChange.bind(this)}/>)
```

Comment:
FYI: I think parenthesis expression is only necessary if the opening tag is the next line after `return` statement. 
```javascript
return <div> ... </div>
```
```javascript
return <div> 
... 
</div>
```
but:
```javascript
return (
  <div> 
  ...
  </div>
)
```